test cases/unit/101 python without pkgconfig/meson.build

```diff
@@ -0,0 +1,4 @@
+project('python wihtout pkgconfig', 'c')
```

Comment:
`wihtout` -> `without`

Reply:
This was already merged into master at the time of this comment.